analysed-packages/imx-gpu-viv/version-6.4.3.p4.6/imx-gpu-viv-6.4.3.p4.6-SPDX2TV.spdx

```diff
@@ -0,0 +1,3000 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_imx-gpu-viv-6.4.3.p4.6-aarch32.tar.gz_1725442449.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Sascha Dierberg
+CreatorComment: <text>
+This document was created using license information and a generator from Fossology.
+It contains the license and copyright analysis of imx-gpu-viv 6.4.3.p4.6.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2024-09-04T09:34:10Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: imx-gpu-viv-6.4.3.p4.6-aarch32.tar.gz
+PackageFileName: imx-gpu-viv-6.4.3.p4.6-aarch32.tar.gz
+SPDXID: SPDXRef-upload502
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: f28bedbb380f05d59f876086d7bb8bc8f5fe7ddf
+PackageChecksum: SHA1: bfa30aa1cfaab1141cf4ca01d0afbb4d30042159
+PackageChecksum: SHA256: d961eeaac19c66690b41a191c729a7b43d66202a24f83b421324431f20d0c227
+PackageChecksum: MD5: 28c7b4ffc0ebf5efabda0b37d0cccc63
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: NOASSERTION
+PackageLicenseComments: <text> licenseInfoInFile determined by Scanners:
+ - nomos ("4.4.0".903df0)
+ - monk ("4.4.0".903df0)
+ - ojo ("4.4.0".903df0)
+ - scancode ("4.4.0".903df0) </text>
+PackageLicenseInfoFromFiles: NOASSERTION
+PackageCopyrightText: NOASSERTION
+
+Relationship: SPDXRef-DOCUMENT DESCRIBES SPDXRef-upload502
+
+
+##--------------------------
+## File Information
+##--------------------------
+
+##File
+
+FileName: imx-gpu-viv-6.4.3.p4.6-aarch32.tar.gz/imx-gpu-viv-6.4.3.p4.6-aarch32.tar/imx-gpu-viv-6.4.3.p4.6-aarch32/gpu-core/usr/include/HAL/gc_hal_enum.h
```

Comment:
I package I analysed (see comment in the README.file contains the following file for example:
kernel-module-imx-gpu-viv-upstream-6.4.3.p4.6.zip/kernel-module-imx-gpu-viv-upstream-6.4.3.p4.6/kernel-module-imx-gpu-viv-src/hal/kernel/inc/gc_hal_enum.h
which is a total different path and the license of the file I downloaded is GPL-2.0-or-later OR MIT